apps/preview/nuxt/pages/showcases/Footer/FooterBasic.vue

```diff
@@ -0,0 +1,236 @@
+<template>
+  <div class="pt-10 bg-neutral-100">
+    <div
+      class="grid justify-center grid-cols-[1fr_1fr] md:grid-cols-[repeat(4,1fr)] px-4 md:px-6 pb-10 max-w-[1536px] mx-auto"
+    >
+      <div v-for="{ label, subcategories } in categories" :key="label" class="min-w-[25%] xs:min-w-[50%] xs:pb-4">
+        <div class="text-lg font-medium leading-7 text-neutral-900 font-body">{{ label }}</div>
+        <p
+          v-for="{ subcategoryLabel, link } in subcategories"
+          :key="subcategoryLabel"
+          class="py-2 typography-text-sm font-body"
+        >
+          <SfLink
+            class="no-underline text-neutral-600 hover:underline hover:!text-neutral-900 active:underline active:!text-neutral-900"
+            variant="secondary"
+            :href="link"
+          >
+            {{ subcategoryLabel }}
+          </SfLink>
+        </p>
+      </div>
+    </div>
+    <hr />
+    <div class="py-10 md:flex md:mx-auto max-w-[1536px]">
+      <div v-for="{ label, icon, link, details } in contactOptions" :key="label" class="mx-auto my-4 text-center">
+        <component :is="icon" size="lg" />
+        <p class="py-1 my-2 font-medium typography-text-lg font-body">
+          <SfLink
+            variant="secondary"
+            class="no-underline text-neutral-600 hover:underline hover:!text-neutral-900 active:underline active:!text-neutral-900"
+            :href="link"
+          >
+            {{ label }}
+          </SfLink>
+        </p>
+        <p v-for="option in details" :key="option" class="text-sm leading-5 text-neutral-600 font-body">
+          {{ option }}
+        </p>
+      </div>
+    </div>
+    <div class="bg-neutral-900 justify-end px-4 py-10 md:flex md:py-6 max-w-[1536px] mx-auto">
+      <div class="flex justify-center py-2 gap-x-4 md:self-start">
+        <SfButton
```

Comment:
On figma there is one padding for top, bottom, left and right (I think square props should work for it)